docs/UserGuide.md

```diff
@@ -173,75 +203,104 @@ Format: `give INDEX d/DESCRIPTION by/ D/M/YYYY [,HHMM]`
+</div>
 
 Examples:
-* `find n/Bernice` returns the students with name `Bernice`. 
+* `find n/Bernice` returns the students with name of `Bernice`. 
 * `find m/CS1101S` returns the students with module `CS1101S`.
 * `find t/Lab15` returns the students with tutorial `Lab15`.
-* `find n/Bernice m/MA1521` returns the students with name `Bernice` or study module `MA1521`.
+* `find n/Bernice m/MA1521` returns the students with name of `Bernice` or study module `MA1521`.
 * `find m/CM1417 t/Group04` returns the students with module `CM1417` or in tutorial `Group04`.
-* `find n/Evian m/CS2103T t/Group10` returns the students who at least satisfy one of requirements: with name `Evian`, study module`CS2103T` and in tutorial `Group10`.
-* `find t/E34 n/Brian m/GEQ1000` returns the students who at least satisfy one of requirements: with name `Brian`, study module`GEQ1000` and in tutorial `E34`.
-* `find n/alex david m/cs1231 cs2103t` returns the students who at least satisfy one of requirements: with name contains `david` or `alex`, study module`cs1231` or `cs2103t`.
+* `find n/Evian m/CS2103T t/Group10` returns the students who at least satisfy one of the requirements: with name of `Evian`, study module`CS2103T` and in tutorial `Group10`.
+* `find t/E34 n/Brian m/GEQ1000` returns the students who at least satisfy one of the requirements: with name of `Brian`, study module`GEQ1000` and in tutorial `E34`.
+* `find n/alex david m/cs1231 cs2103t` returns the students who at least satisfy one of the requirements: `david` or `alex` or both can be part of their names seperated by space, study module`cs1231` or `cs2103t`.
 
 
 ![result for 'find Example'](images/userguide/findExample.png)
 
-### Deleting an assignment with deadline : `remove`
+### Deleting an assignment with a deadline: `remove`
 
 Removes the specified assignment from the displayed assignment list.
 
 Format: `remove INDEX`
 
-* Removes the assignment at `INDEX` in current displayed assignment list of a student.
+* Removes the assignment at `INDEX` in the currently displayed assignment list of a student.
 
 Examples:
 * `remove 10` deletes the 10th assignment in the displayed assignment list
 
-### Marking an assignment : `done`
+### Marking an assignment: `done`
 
 Marks a specified assignment's deadline of a student as completed.
-* Assignments with uncompleted/pending status will have an orange tag.
-* Assignments with completed status will have a green tag.
-* Only works if current displayed assignment list is non-empty.
 
 Format: `done INDEX`
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:**<br>
+
+* Assignments with pending status will have an orange tag.
+
+* Assignments with completed status will have a green tag.
+
+* Only works if currently displayed assignment list is not empty.
+
+</div>
+
+<div markdown="span" class="alert alert-warning">
+:exclamation:**Caution:**
+
+* You can not mark the already completed assignment as done any more.
```

Comment:
I think we can omit this? 
Firstly, I think its not at level of caution
Secondly, I think this might be common sense to users

Reply:
yes,just realized

Reply:
done